docs/using/installing/index.rst

```diff
@@ -492,14 +492,92 @@ The ZFS ``pool`` variable should match the pool name created in the ZFS section.
    "control-service":
       "hostname": "${CONTROL_NODE}"
       "port": 4524
+
+``${CONTROL_NODE}`` should be replaced with the address of the control node.
+The optional ``port`` variable is the port on the control node to connect to.
+This value must agree with the configuration for the control service telling it on what port to listen.
```

Comment:
Probably should make it clearer that simply omitting it is the easiest thing to do and will just work.